### ISSUE TITLE
Bug in code causing lambda error

### ISSUE THREAD

**Issue by ashishpujari** [awslabs/aws-config-rdk#186]
There appears to be a bug in the code in the below file on line 208
`rdk/template/runtime/python3.7/rule_code.py`

The code on line 208 is
```
configuration_item_summary = check_defined(invoking_event['configuration_item_summary'], 'configurationItemSummary')
```

The code should be
```
configuration_item_summary = check_defined(invoking_event['configurationItemSummary'], 'configurationItemSummary')
```

**Comment by jack1902** [comment]
Do you have a stack trace from your lambda?

**Comment by ashishpujari** [comment]
I don't have it now. But it was to the effect of "Key Error : the key invoking_event['configuration_item_summary'] does not exist"

**Comment by ashishpujari** [comment]
If you create a rule which has a config item change trigger, you should see the key error too in the logs once the rule is triggered..

**Comment by jack1902** [comment]
ok so from the docs here: 

https://docs.aws.amazon.com/config/latest/developerguide/oversized-notification-example.html

i can see that you are right, i can go ahead and open a pr to fix this issue, but i can't actually merge it as i don't have any rights on this repo

**Comment by jack1902** [comment]
@ashishpujari Opened a PR to fix the issue within the templates for 3.6 and 3.7. you can see from the link above that this is the event passed in. Also looking at the templates for 2.7 and 3.6-managed, they have the correct dictionary lookup

**Comment by mbeacom** [comment]
Resolved 3 years ago by https://github.com/awslabs/aws-config-rdk/pull/197
